modules/ibi-create-config-iso.adoc

```diff
@@ -311,4 +311,14 @@ target-0   HostValidationSucceeded   ClusterInstallationSucceeded  ibi-bmh
 [WARNING]
 ====
 If the `ImageClusterInstall` resource is deleted, the IBI Operator reattaches the `BareMetalHost` resource and reboots the machine.
-====
\ No newline at end of file
+====
+
+. When the installation completes, you can retrieve the `kubeconfig` secret to log in to the managed cluster by running the following command:
++
+[source,terminal]
+----
+$ oc extract secret/<cluster_name>-admin-kubeconfig -n <cluster_name>  --to - > <directory>/<cluster_name>-kubeconfig
```

Comment:
Looks good, it's what we use in CI
https://github.com/openshift/release/blob/ca576bff5d40301a5ff79f6c1f942117798ad3a5/ci-operator/step-registry/image-based-install-operator/image-based-install-operator-commands.sh#L111

Just  nit
```suggestion
$ oc extract secret/<cluster_name>-admin-kubeconfig -n <cluster_namespace>  --to - > <directory>/<cluster_name>-kubeconfig
```
Early in the doc the namespace is `ibi-ns` while cluster_name is `ibi-cluster`

Reply:
Thanks, updated with your suggested change